src/test/java/org/mule/extensions/jms/test/connection/JmsReconnectionTestCase.java

```diff
@@ -109,7 +109,7 @@ private void startFlow(String flowName) throws Exception {
   @Step("Restart JMS broker")
   private void restartBroker() throws InterruptedException {
     amqBroker.stop();
-    Thread.sleep(500);
+    Thread.sleep(3000);
```

Comment:
Did you try it with a smaller value? Remember that the overall idea is for the tests to take the less time possible.

I think the change from 500 to 3000 is quite a considerable jump. Did you try for example with 1000 or 2000 ms? 

Anyways, I'd prefer to wait for @aiannucci 's  criteria on this.

Reply:
Does this mean that it takes longer in Windows than in other OSs? I wouldn't be surprised, but just to confirm

Reply:
I tried it with several values.
- It was enough with 2000ms to execute only the JmsReconnectionTestCase.
- It was enough with 3000ms to run all the tests of the jJMS-CONNECTOR and that it would not fail.
- And it takes more time in Windows to restart JMS broker.